src/plugins/data_explorer/public/plugin.ts

```diff
@@ -105,6 +107,7 @@ export class DataExplorerPlugin
             ...withNotifyOnErrors(coreStart.notifications.toasts),
           }),
           viewRegistry: viewService.start(),
+          discoverActions: this.discoverActions,
```

Comment:
Nit: maybe `actions` is good enough as we are already under the context of discover.